.github/workflows/continuous-integration.yaml

```diff
@@ -12,10 +12,19 @@ jobs:
         uses: 'shivammathur/setup-php@v2'
         with:
           php-version: '7.4'
+          coverage: 'none'
           extensions: 'json, mbstring, tokenizer'
-          tools: 'cs2pr, php-cs-fixer:2.16.4'
```

Comment:
Removing `cs2pr` as it doesn't bring much value with php-cs-fixer as it doesn't indicate line numbers